src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpInferredMemberNameReducer.Rewriter.cs

```diff
@@ -58,7 +57,6 @@ public override SyntaxNode VisitArgument(ArgumentSyntax node)
                 {
                     return SimplifyNode(
                         node,
-                        parentNode: node.Parent,
```

Comment:
passing `node.parent` is the default behavior for non-customized nodes, so just calling the core helper is what we want.  subtypes should not override this behavior.